docs/changelog/1_0_a2.rst

```diff
@@ -0,0 +1,475 @@
+===========
+1.0 Alpha 2
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 2.
+
+
+JavaScript Driver
+=================
+
+EdgeDB has a new high-performance native
+`EdgeDB driver <https://github.com/edgedb/edgedb-js>`_ for NodeJS 10+.
+
+It is written in strict TypeScript, thouroughly tested, and has a
+first-class async/await support.  Not to mention it is at least **2x faster**
+than comparable current PostgreSQL JavaScript drivers.
+
+Install it via ``npm`` or ``yarn``:
+
+.. code-block:: bash
+
+   $ npm install edgedb
+
+and it is ready for use:
+
+.. code-block:: javascript
+
+    const edgedb = require("edgedb");
+
+    async function main() {
+      const conn = await edgedb.connect({
+        user: "edgedb",
+        host: "127.0.0.1",
+      });
+
+      try {
+        console.log(await conn.fetchOne("SELECT 1 + 1"));
+      } finally {
+        await conn.close();
+      }
+    }
+
+    main();
+
+The documentation can be found `here </docs/clients/01_js/index>`_.
+
+
+Standard Library
+================
+
+std::bigint
+-----------
+
+The new :eql:type:`std::bigint` scalar type is an arbitrary precision integral
+type.  The motivation for the new type is that many platforms lack a true
+multi-precision decimal type, but implement an arbitrary-precision integer
+type (JavaScript is a prominent example).  The decimal literals syntax
+was updated:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT 1n IS std::bigint;
+    {true}
+
+    db> SELECT 1.0n IS std::decimal;
+    {true}
+
+The :eql:type:`std::bigint` and :eql:type:`std::decimal` maintain the
+relationship that is similar to the relationship between :eql:type:`std::int64`
+and :eql:type:`std::float64`.  All sized integer types are implicitly
+castable to bigint, and bigint itself can implicitly cast to
+:eql:type:`std::decimal`.
+
+
+New "cal" Module
+----------------
+
+Non-timezone aware date/time types and functions are moved into the new
+``cal`` module.  This new separation promotes :eql:type:`std::datetime`
+to be the default safe choice for most use cases.  The types in the ``cal::``
+module are useful to implement calendars, alarms, reminders, etc, but have
```

Comment:
"etc" should always be followed by a trailing period. Hence, "etc., but ..."
([1.](https://english.stackexchange.com/questions/188927/can-we-use-the-abbreviation-etc-without-a-period-full-stop) [2.](https://www.editage.com/insights/research-paper-comma-after-etc-at-the-end-of-a-sentence))